### ISSUE TITLE
Rebound on walking into bamboo (and probably others) with `"offset_type": "none"` when playing on unmodded server

### ISSUE THREAD

**Issue by helpimnotdrowning** [MehVahdJukaar/polytone#117]
I use the [Golden Days](https://github.com/PoeticRainbow/golden-days) (base) pack with Polytone, which, among other plant-type blocks, sets bamboo ([`assets/minecraft/polytone/block_properties/bamboo.json`](https://github.com/PoeticRainbow/golden-days/blob/master/golden-days-base/assets/minecraft/polytone/block_properties/bamboo.json)) to
```json
{
  "offset_type": "none"
}
```

This works perfectly fine on a singleplayer world, where the client and server seem aware of the change to the block hitbox, and no rebound happens:
![image](https://github.com/user-attachments/assets/7d221de5-841a-4a06-bab3-c85b6704a031)

https://github.com/user-attachments/assets/a524b374-0503-4646-9627-58a2a493940b

My Purpur multiplayer server, however, has no knowledge of this change, and rebounds me when I try to walk into where the bamboo is *supposed* to be:

Without Golden Days:

https://github.com/user-attachments/assets/700a06a5-62e1-406d-9c28-578b644859ad

With:

https://github.com/user-attachments/assets/b9f9156e-00e1-4b71-9383-a1f17da4a1cd

Ideally, Polytones should detect that the server doesn't recognize the same bounding box that it does and override the resource pack

**Comment by MehVahdJukaar** [comment]
Yeah that change is purely visual so while on single player it also is used for server side logic as there's just one copy of that into there (untill Mojang adds Datapack blocks) on server the collision logic is different. Bamboo is the only block that moves it's entire hitbix based off offset

**Comment by MehVahdJukaar** [comment]
So solution here is hard. I could as you say ignore the offset type when not in single player so the two match

**Comment by MehVahdJukaar** [comment]
Or I could make blocks hold a second offset type just used for rendering

**Comment by MehVahdJukaar** [comment]
next version will have the offset just apply to the visuals

**Comment by MehVahdJukaar** [comment]
try new version an see if it works better
